phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<phpunit colors="true">
```

Comment:
The bootstrap is required for travis. moreover, we want people to be able to run the tests by just running phpunit from the root - it should be IDE agnostic.

For the bundle structure, I agree, but let's do that in another PR after this is merged.